app.js

```diff
@@ -1,8 +1,9 @@
 const express = require('express')
+const bodyParser = require('body-parser')
 
 const expressLayouts = require('express-ejs-layouts')
 const ejs = require('ejs')
-const acolhido = require('./public/js/acolhido.js')
+// const acolhido = require('./public/js/acolhido.js')
```

Comment:
Código comentado? 
![Judging you](https://www.awesomelyluvvie.com/wp-content/uploads/2012/11/JudgingYou.gif)